api/routes.go

```diff
@@ -17,7 +17,7 @@ func RegisterRoutes(r *mux.Router) {
 	// PUT Endpoints
 
 	// PATCH Endpoints
-
+	r.HandleFunc("/api/v1/changepass/{newPassword:}", endpoints.ChangePassword).Methods("PATCH")
```

Comment:
Consider changing into something more secure in future

Reply:
remvoed parameter in recent commit. New password will be extracted from json body